Wabbajack.App.Blazor/wwwroot/index.html

```diff
@@ -32,12 +34,19 @@
 <div id="app"></div>
 
 <script src="_framework/blazor.webview.js"></script>
+<script src="_content/Blazored.Modal/blazored.modal.js"></script>
 <script>
     async function setImageUsingStreaming(imageElementId, imageStream) {
         const arrayBuffer = await imageStream.arrayBuffer();
         const blob = new Blob([arrayBuffer]);
         document.getElementById(imageElementId).src = URL.createObjectURL(blob);
     }
```

Comment:
Are we still using this somewhere? I saw `getBlobUrlFromStream` so this can probably be deleted.